pom.xml

```
@@ -435,6 +435,16 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M5</version>
+        <configuration>
+          <systemPropertyVariables>
+            <java.io.tmpdir>${user.dir}/target</java.io.tmpdir>
+          </systemPropertyVariables>
```

I tried this, but it didn't work for me. However, the following did work for me. From what I've read, I suspect it has something to do with the fact that surefire forks. Maybe it doesn't propagate the system properties to the forked executions?

```suggestion
          <argLine>-Djava.io.tmpdir=${project.build.directory}</argLine>
```

However, I suspect this might break if there's a space in the path to the project.

---

Quoting it should solve any issues with spacing - see the updated revision

---

Okay, we'll go with that.